test/MainnetForkUSDC.ts

```diff
@@ -7,27 +7,27 @@ import setupUSDC, { USDC_PRECISION } from "./helpers/usdc";
 const { parseUnits } = ethers.utils;
 
 describe("MainnetForkUSDC", function () {
-  xit("gives 1,000,000 USDC to first signer", async function () {
-    const { usdc, signerAddresses } = await loadFixture(setupUSDC);
```

Comment:
Let's remove the unused imports where we are no longer using loadFixture

Reply:
removed all unused imports